app/src/main/res/layout/activity_main.xml

```diff
@@ -27,6 +27,16 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/server_button" />
 
+    <EditText
+        android:id="@+id/input_username"
+        android:layout_height="wrap_content"
+        android:layout_width="match_parent"
+        android:inputType="text"
+        android:hint="Enter username"
```

Comment:
this is fine for testing and getting things running (so its basically fine for now) but if you put this constant string in strings.xml we get rid of a smell

dominik might not see this as important though since he did that a lot ^^

Reply:
Yeah u got me. It's bad practice to use hardcoded strings (ide warns for a reason). I thought as long as I don't know what text we should really display....
Sonar gives us the tool that we can replace all of them in one step. With all languages we know.
@enakecanovic I think you are capable of an extra language, maybe we should think about adding that translation as well?

Reply:
I really like the idea of putting an extra language there, free code lines aswell btw.